scripts/describe-sboms.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
```

Comment:
(perhaps we should describe what this script intends to do better - though I realize I haven't set a good example for that in the other scripts yet either ;) )